<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWindow</class>
 <widget class="QWidget" name="HistoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Time Range</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="fromDateEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="fromButton">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="toDateEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="toButton">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lastWeekButton">
        <property name="text">
         <string>Last week</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lastMonthButton">
        <property name="text">
         <string>Last month</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="executeButton">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>234</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="historyTreeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Song</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Distribution</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="distributionTreeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Day</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Play counts</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Top Songs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="topSongsTreeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Song</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Play counts</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Top Artists</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTreeWidget" name="topArtistsTreeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Artist</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Play counts</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Top Genres</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTreeWidget" name="topGenresTreeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Genre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Play counts</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HistoryWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>750</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
